backend/src/main/java/com/example/backend/controller/WorkoutController.java

```diff
@@ -25,7 +26,7 @@ public Workout addWorkout(@RequestBody Workout workout) {
         return workoutService.addWorkout(workout);
     }
 
-    @GetMapping("{id}")
+    @GetMapping("/workouts/{id}")
```

Comment:
You could add die / workouts in line 12 so you can leave it out in the other mapping